client/js/widgets/widget.js

```diff
@@ -364,11 +370,7 @@ class Widget extends StateManaged {
   }
 
   updateOwner(oldName, newName) {
-    const o = this.p('owner');
-    if(o && o == oldName)
-      this.domElement.classList.add('foreign');
-    if(o && o == newName)
-      this.domElement.classList.remove('foreign');
+    this.domElement.className = this.classes();
```

Comment:
This no longer uses `oldName` nor `newName` parameters. Remove them? Rename it to `updateClasses()`?

Reply:
I don't know, I feel like the caller of that function doesn't know it's handled with classes. The caller wants to tell the widget that the current player changed its name.

Reply:
It seems misleading to have parameters that are unused. Any typical static analysis tool and IDE would flag this.

Reply:
You're right. I removed the parameters. Renaming the function still seems weird to me.